docs/examples/custom_operations/custom_operator/naive_histogram/README.md

```diff
@@ -0,0 +1,48 @@
+# DALI custom operator example
+
+This is an example/template of a DALI Custom operator.
+The implemented operation is a (very) naive histogram on a grayscale image.
+The NaiveHistogram operator is a GPU-only operator, which uses a badly written
+CUDA kernel to calculate the histogram. The purpose here is not to show-case
+how the write CUDA kernels, but to present how to use CUDA kernels within
+DALI operators.
+
+# Running the example
+
+To run the example, please follow the steps:
+
+1. Clone or download the repo
+1. Install DALI wheel, for example using [these instructions](https://docs.nvidia.com/deeplearning/dali/main-user-guide/docs/installation.html#pip-official-releases)
+1. Build the `libnaivehistogram.so`:
+```bash
+$ cd naive_histogram
+$ mkdir build
+$ cd build
+$ cmake ..
+$ make -j
+```
+
+4. **(Optional)** To run the example, you need to provide some test files (actually, images). It can 
+really be any image, as long as it's not broken. The test provides preliminary image list as an 
+example. We're using a [DALI_Extra](https://github.com/NVIDIA/DALI_extra) repository for that
+purpose, but it's not obligatory. Should you want to use DALI Extra too, please clone (or download)
+the repo and set a environment variable defining the path to DALI Extra:
+```bash
+git clone -d /path/to/dali/extra https://github.com/NVIDIA/DALI_extra
+export DALI_EXTRA_PATH=/path/to/dali/extra
```

Comment:
Please mention that DALI_extra requires the `git lfs`.

Reply:
It does not in this case, so I won't clutter these instructions with `git lfs`

Reply:
How does it not require `git lfs`?

Reply:
No idea, they just not...

Reply:
I would insist on mentioning the `git lfs`. The fact that it works with a subset of the repo will lead to the problems when the user wants to use the part that's broken.